File: src/main/kotlin/org/ailingo/server/model/UserDto.java
package org.ailingo.server.model;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.Instant;
import java.util.UUID;

@Getter
@Setter
@NoArgsConstructor
public class UserDto {

    private UUID id;
    private String login;
    private String avatar;
    private int xp;
    private Instant registration;
    private Instant lastLoginAt;

}

File: src/main/kotlin/org/ailingo/server/model/UserDto.kt
package org.ailingo.server.model

import java.time.Instant
import java.util.*

data class UserDto(
    val id: UUID,
    val login: String,
    val avatar: String,
    val xp: Int = 0,
    val registration: Instant,
    val lastLoginAt: Instant
) {

}